### PR TITLE
Create nano.profile

### PR DESCRIPTION
**Pull request by rusty-snake**
See: #2411

Notes: I think it is still to strict.

### CODE REVIEW

File: etc/nano.profile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+apparmor
+caps.drop all
+machine-id
```
Comment (rusty-snake) [inline etc/nano.profile]: Schould we add `ipc-namespace`?

File: etc/nano.profile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-bin nano
+private-cache
+private-dev
+private-etc nanorc
+private-tmp
```
Comment (reinerh) [inline etc/nano.profile]: by default I can't edit files in /tmp or /etc?

Comment (rusty-snake) [reply etc/nano.profile]: right
>  I think it is still to strict.

File: etc/nano.profile [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+noblacklist ${HOME}/.config/nano
+noblacklist ${HOME}/.nanorc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
```
Comment (rusty-snake) [inline etc/nano.profile]: Will be removed because of DOCUMENTS, MUSIC/VIDOES (playlists)

File: etc/nano.profile [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+nosound
+notv
+nou2f
+novideo
+protocol unix
+seccomp
+shell none
+tracelog
+
+disable-mnt
```
Comment (rusty-snake) [inline etc/nano.profile]: Will be removed.

**Comment by glitsj16** [comment]
Please add support for restricted nano, a.k.a. `rnano` in private-bin. Users might have that set as EDITOR/SUDO_EDITOR.

**Comment by glitsj16** [comment]
Another request if you don't mind. To bring this in line with recent changes in all other profiles, would you mind adding `alternatives` to `private-etc` please?

File: etc/nano.profile (hunk 1 above)
Comment (SkewedZeppelin) [reply etc/nano.profile]: won't hurt, cli only

**Comment by rusty-snake** [comment]
@glitsj16, @Fred-Barclay at the moment I'm tend to remove `private-etc`.

**Comment by rusty-snake** [comment]
From my side it can be merged.

**Comment by glitsj16** [comment]
@rusty-snake There's a double include for globals.local in rnano.profile (due to redirection to nano.profile, which already does that). You can just comment it cfr. https://github.com/netblue30/firejail/issues/2006.

**Comment by rusty-snake** [comment]
@glitsj16 Done.

**Comment by glitsj16** [comment]
@rusty-snake Ah, you commented include rnano.local instead of `include globals.local`. It is the latter that causes the double include, as redirecting to nano.profile in the last line already does that. Also, have you decided to keep the profile from editing files under /etc on purpose or is this restriction something you plan to drop? Personally I tend to do quick edits in /etc quite regularly. Which is just me though, I can always override `private-etc` in a local file. Just asking :smile:.

**Comment by rusty-snake** [comment]
@glitsj16 Arrg, I have edit the wrong line. I wanted to comment globals, 😬 😡.

`private-etc`:
  * My idee was to not put it to firefcg.config like vim or emacs => By default there is no symlink.
  * You need root privilegs to edit in /etc.
    1. You are root or you use sudo => You have a Problem.
    2. You use sudoedit (or sudo -e) => than sudo creates a copy in /tmp
  * sudoedit is the better/more secure way

**==> I will leave `private-etc` and add a comment.**

**Comment by Utini2000** [comment]
Although this profile exists, it will now be linked on Arch when running sudo firecfg?

**Comment by rusty-snake** [comment]
If you think you found a bug, open an issue for it.

Also

https://github.com/netblue30/firejail/blob/e8b693c814a4e157bcae6f5c60d1624d185d8837/src/man/firecfg.1.in#L142-L162

**Comment by Utini2000** [comment]
> If you think you found a bug, open an issue for it.

Yep sorry - I think this is not a bug with this profile but generally with my setup. 
I have several application that are not run in firejail although a profile exists. 
I will keep this problem in https://github.com/netblue30/firejail/discussions/6438

Thanks!

**Pull request merged**